.golangci.yml

```diff
@@ -10,28 +10,23 @@ output:
 linters:
   enable-all: true
   fast: false
-  
-  disable:
-    # These linters are all deprecated. We disable them explicitly to avoid the
-    # linter logging deprecation warnings.
-    - execinquery
 
+  disable:
     # These are linters we'd like to enable, but that will be labor intensive to
     # make existing code compliant.
     - wrapcheck
     - varnamelen
     - testpackage
     - paralleltest
     - nilnil
-    - gomnd
```

Comment:
removed because of deprecation warnings:

`            +go-lint *failed* | level=warning msg="[lintersdb] The linter \"gomnd\" is deprecated (step 2) and deactivated. It should be removed from the list of disabled linters. https://golangci-lint.run/product/roadmap/#linter-deprecation-cycle"`